ui/src/organizations/components/CollectorRow.tsx

```diff
@@ -61,13 +62,7 @@ export default class CollectorRow extends PureComponent<Props> {
               <Button
                 size={ComponentSize.ExtraSmall}
                 color={ComponentColor.Secondary}
-                text={'View'}
-                onClick={this.handleOpenConfig}
-              />
-              <Button
-                size={ComponentSize.ExtraSmall}
-                color={ComponentColor.Secondary}
-                text={'Setup Details'}
+                text={'Setup Instructions'}
```

Comment:
```suggestion
                text="Setup Instructions"
```

Reply:
I posted in #chronograf-code about this, I could go either way on the pattern

Reply:
ahh okay, I missed that. will review